### DISCUSSION TITLE
Button style anchor?

### DISCUSSION THREAD

**Discussion by jweintraub** [twilio-labs/paste#1759] [Q&A] [answered]
A while ago, I saw that DSYS was working on a button-style anchor with a border and arrow. Is that still planned?

**Comment by richbachman** [comment, marked as answer]
Hi @jweintraub 

We have the `as="a"` prop for Button that renders the button as an anchor, so you get all the button styles. Here's an example: https://paste.twilio.design/patterns/button-vs-anchor#buttons-with-link-functionality

Is that what you're looking for?

**Reply by jweintraub** [reply]
Yes! thanks!!
